README.md

```diff
@@ -99,16 +99,22 @@ Following table describes subroutine name and recognizing scope:
 
 ### Annotation
 
-For some reasons, the subroutine name could not be changed. Then, if you apply a hint of scope on annotation, `falco` also understands scope:
+For some reasons, the subroutine name could not be changed. Or you want to use this function in multiple scopes. Multiple scopes
+are declared as comma seperated values.
+
+Then, if you apply a hint of scope on annotation, `falco` also understands scope:
 
 ```vcl
-// @recv
-sub custom_process { // subroutine has `recv` annotation, lint with RECV scope
+// @recv, miss
```

Comment:
I mean annotation name like `@scope:` would be nice so I expected change with:

```
// @scope: recv,miss
```

This suggestion means we implement _named-annotation_ for additional features in the future.

Reply:
I see what you are saying but I am worried that this is not back compat, so it will break people when they update.

Reply:
Could both the `@scope:` and single `@recv`/`@fetch` etc annotations both be supported for now? Maybe the latter style can be deprecated and support will be dropped in favour of `@scope` in the future.

Reply:
Yeah I could do it as an addition but it might be confusing. Ideally we could have a v2 ideas issue were we append breaking changes and when enough things are added we can have a `--v2` flag

Reply:
added `@scope` as an addition as well